srml/system/src/lib.rs

```diff
@@ -317,6 +316,8 @@ decl_storage! {
 		AllExtrinsicsWeight: Option<u32>;
 		/// Map of block numbers to block hashes.
 		pub BlockHash get(block_hash) build(|_| vec![(T::BlockNumber::zero(), hash69())]): map T::BlockNumber => T::Hash;
+		/// Maximum number of block number to block hash mappings to keep (oldest pruned first).
+		BlockHashCount get(block_hash_count) config(): T::BlockNumber = 250.into();
```

Comment:
Sounds like something that should be `parameter_type!`.

Reply:
I don't know what `parameter_type!` is, but won't this always be defined in `T::BlockNumber`?

Reply:
Just look into `runtime` and search for it. You require to always look into the storage for a value that probably never changes. Using `parameter_types!`, the value is hard coded and compiled into the runtime.